packages/playground/src/views/nodes.vue

```diff
@@ -160,6 +160,9 @@ export default {
           if (count) {
             nodesCount.value = count;
           }
+          if (count == 0) {
+            nodesCount.value = 0;
+          }
```

Comment:
I'm worry about `NaN` issue if the count is undefined maybe 
so we can use `??`
```suggestion
            nodesCount.value = count ?? 0
```